doc/integrator/c2cwsgiutils.rst

```diff
@@ -23,26 +23,24 @@ To have basic authentication of those tools (and on tile generation) you can set
 You can also activate the authentication on GitHub with OAuth2, the configuration is in the
 ``env.project`` file:
 
-* Verify that the ``C2C_AUTH_GITHUB_REPOSITORY`` is correctly configured.
+* Verify that the ``C2C_AUTH_GITHUB_REPOSITORY`` is correctly configured to the project repository.
```

Comment:
suggestion:
to the project repository -> to match the project repository name on GitHub.